ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ClusterSelection.tsx

```diff
@@ -149,9 +161,19 @@ function ClusterSelection({ clusters, isFetchingClusters }: ClusterSelectionProp
                 </Flex>
             </PageSection>
             <Divider component="div" />
-            <Form className="pf-u-py-lg pf-u-px-lg">
+            <Form className="pf-u-py-lg pf-u-px-lg" ref={alertRef}>
+                {formikTouched.clusters && formikValues.clusters.length === 0 && (
+                    <Alert
+                        title="At least one cluster is required to proceed"
+                        variant="danger"
```

Comment:
A question we might ask in next UI/UX meeting is danger versus warning.
* Plus for danger here: red color is conventional for validation errors
* Minus for danger here: failure to create or update seem more like what guidelines are thinking:

    https://www.patternfly.org/design-foundations/icons
    ExclamationCircleIcon which I think the `Alert` element renders:
    > Represents alert status: danger, major error or critical error

Reply:
Thats a good question. I added it to the UX/UI sync agenda.